### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.product-cols__subheading.no-link)

### ISSUE THREAD

**Issue by A11Ybot-OP** [firebase/friendlyeats-web#218]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Firebase](https://firebase.google.com/)
#### Element path
.product-cols__col:nth-child(1) > .product-cols__header > .product-cols__subheading.no-link
#### Snippet
`<p class="product-cols__subheading no-link">Accelerate app development with fully managed backend infrastructure
</p>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.37 (foreground color: #757575, background color: #f8f9fa, font size: 13.5pt (18px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 98.0.4758.102

**Comment by A11Ybot-OP** [comment]
there are 17 other similar colour-contrast issues
